pulsar-broker/src/main/java/com/yahoo/pulsar/broker/service/BrokerService.java

```diff
@@ -824,4 +841,76 @@ public AuthenticationService getAuthenticationService() {
     public List<PersistentTopic> getAllTopicsFromNamespaceBundle(String namespace, String bundle) {
         return multiLayerTopicsMap.get(namespace).get(bundle).values();
     }
+    
+    public ZooKeeperDataCache<Map<String, String>> getDynamicConfigurationCache() {
+        return dynamicConfigurationCache;
+    }
+
+    /**
+     * Update dynamic-ServiceConfiguration with value present into zk-configuraiton-map and register listeners on
+     * dynamic-ServiceConfiguration field to take appropriate action on change of zk-configuraiton-map.
+     */
+    private void updateConfigurationAndRegisterListeners() {
+        // update ServiceConfiguration value by reading zk-configuration-map
+        updateDynamicServiceConfiguration();
+        // update brokerShutdownTimeoutMs value on listener notification
+        registerConfigurationListener("brokerShutdownTimeoutMs", new Consumer<String>() {
+            @Override
+            public void accept(String shutdownTime) {
```

Comment:
addressed it on [commit](https://github.com/yahoo/pulsar/pull/186/commits/f4adac9c376b63e1e7e4850cacefde29e8ed8481) can you please take a look of it.